configs/_base_/models/biggan_32x32.py

```diff
@@ -0,0 +1,29 @@
+model = dict(
+    type='BasiccGAN',
+    generator=dict(
```

Comment:
`num_classes` should be set for `BasiccGAN` as well.
Otherwise, `AttributeError` would be raised in `conditional_demo`

Reply:
I found no error report in `conditional_demo` with the original config. However, I still change this as you said.

Reply:
Have you pulled the latest master branch?

Reply:
Such error would only take place when using `sample-all-classes` argument 😆